case-lib/pipeline.sh

```diff
@@ -46,8 +46,10 @@ func_pipeline_export()
     fi
 
     local opt=""
-    # acquire filter option
-    [[ "$2" ]] && opt="-f '$2'"
+    [ -z "$2" ] || opt="$2"
+    # In no HDMI mode, exclude HDMI pipelines
+    [ -z "$NO_HDMI_MODE" ] || opt="$opt & ~pcm:HDMI"
+    opt="-f '${opt}'"
```

Comment:
Did you test `-f & ~pcm:HDMI` and does it work?

Reply:
`-f & ~pcm:HDMI` doens't work, but it is not possible to happen, because https://github.com/thesofproject/sof-test/blob/5af711fcab5f94ff430a5a11f9f742f071129833/case-lib/pipeline.sh#L14

$2 always gets a value.

Reply:
Great, then remove the misleading `[ -z "$2" ]` test.

Reply:
All review addressed, thanks @marc-hb. I will submit a cleanup PR for config.sh next.